cluster/network.go

```diff
@@ -27,30 +28,36 @@ const (
 	CPPortListenContainer     = "rke-cp-port-listener"
 	WorkerPortListenContainer = "rke-worker-port-listener"
 
-	KubeAPIPort         = "6443"
-	EtcdPort1           = "2379"
-	EtcdPort2           = "2380"
-	ScedulerPort        = "10251"
-	ControllerPort      = "10252"
-	KubeletPort         = "10250"
-	KubeProxyPort       = "10256"
-	FlannetVXLANPortUDP = "8472"
+	KubeAPIPort      = "6443"
+	EtcdPort1        = "2379"
+	EtcdPort2        = "2380"
+	ScedulerPort     = "10251"
+	ControllerPort   = "10252"
+	KubeletPort      = "10250"
+	KubeProxyPort    = "10256"
+	FlannelVxLanPort = 8472
+
+	FlannelVxLanNetworkIdentify = 1
 
 	ProtocolTCP = "TCP"
 	ProtocolUDP = "UDP"
 
 	NoNetworkPlugin = "none"
 
-	FlannelNetworkPlugin = "flannel"
-	FlannelIface         = "flannel_iface"
-	FlannelBackendType   = "flannel_backend_type"
+	FlannelNetworkPlugin               = "flannel"
+	FlannelIface                       = "flannel_iface"
+	FlannelBackendType                 = "flannel_backend_type"
+	FlannelBackendPort                 = "flannel_backend_port"
+	FlannelBackendVxLanNetworkIdentify = "flannel_backend_vni"
```

Comment:
The `VNI` parameter is only used in windows flannel. Maybe we need to comment it here.